src/NSubstitute.Analyzers.CSharp/NSubstitute.Analyzers.CSharp.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
@bdovaz thanks for the PR. Did you have a chance to publish nuget package locally and test if it still works in VS or Rider?

Reply:
Yes, with Unity it works perfectly